<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>calci</class>
 <widget class="QMainWindow" name="calci">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>calci</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="display">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>150</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>721</width>
      <height>80</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="add">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #FF8C00;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="subtract">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #FF8C00;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>sub</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="multiply">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #FF8C00;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>mul</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="divide">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #FF8C00;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>div</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>721</width>
      <height>217</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="1">
      <widget class="QPushButton" name="memory">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>+/-</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="button6">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>6</string>
       </property>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QPushButton" name="button9">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>9</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="button5">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>5</string>
       </property>
      </widget>
     </item>
     <item row="4" column="2">
      <widget class="QPushButton" name="equal">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>equal</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <widget class="QPushButton" name="button1">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="button7">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>7</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="button2">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>2</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0">
      <widget class="QPushButton" name="memoryplus">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>Memory+</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2">
      <widget class="QPushButton" name="memoryminus">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>Memory-</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QPushButton" name="button0">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="button3">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>3</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="button4">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>4</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QPushButton" name="clear">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>clear</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QPushButton" name="button8">
       <property name="styleSheet">
        <string notr="true">QPushButton{
background-color: #C0C0C0;
border: 1px solidgray;
padding :5px
}

QPushButtonpressed{
background-color: #A9A9A9;
border: 1px solidgray;
padding :5px
}</string>
       </property>
       <property name="text">
        <string>8</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLineEdit" name="display2">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>90</y>
      <width>71</width>
      <height>51</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="display3">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>20</y>
      <width>71</width>
      <height>61</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="display4">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>20</y>
      <width>91</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
